projects/UIP.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: project
+title: UIP
+permalink: projects/UIP
+logo: "https://raw.githubusercontent.com/NIT-dgp/UIP-misc/master/artwork/UIP_logo_v4.png"
+description: A cross-platfrom desktop application for wallpapers.
```

Comment:
I intentionally left it that way, don't mind, please @abhsag24. SEO optimization, the query `scheduling wallpapers` seems too less to be made, when compared to `application for wallpapers`.